core/src/style/textStyle.cpp

```diff
@@ -111,6 +111,18 @@ auto TextStyle::applyRule(const DrawRule& _rule, const Properties& _props) const
         }
     }
 
+    if (_rule.get(StyleParamKey::repeat_group, p.labelOptions.repeatGroup)) {
+        StyleParam::Width repeatDistance;
+        if (_rule.get(StyleParamKey::repeat_distance, repeatDistance)) {
+            p.labelOptions.repeatDistance = repeatDistance.value * m_pixelScale;
+        }
+    } else {
+        // TODO: default to 'draw.key'
+    }
+
+    // TBD: should avoid allocation for combined string
+    p.labelOptions.repeatGroup += "/" + p.text;
+
```

Comment:
If the label text need to be kept in TextLabel like this, I think it would be better to have separate members for repeatGroup and text. This will also increase the chance to avoid heap allocated string when both fit into 'small-strings' (16 or 24 chars)